README.md

```diff
@@ -271,15 +271,30 @@ See
 [Defining your own grouping derivatives](#defining-your-own-grouping-derivatives)
 below for details on how to add your own grouping derivatives.
 
+The `groupedAggregates` field accepts a few arguments in addition to `groupBy`:
+
+- `orderBy` – controls how groups are sorted. You can order by any aggregate that
+  appears in the grouped output (e.g. `SUM_POINTS_DESC`).  
+- `first` / `last` – slice the ordered groups, returning only the leading or
+  trailing `n` groups.  
+
+Always pair `first` or `last` with an explicit `orderBy` so PostgreSQL can
+deterministically rank the groups before trimming them.
```

Comment:
Suggest that we remove `last` - if they need that they should do the inverse order.